plugins/services/src/js/stores/MarathonStore.js

```diff
@@ -474,6 +475,14 @@ class MarathonStore extends GetSetBaseStore {
         item["images"] = CosmosPackagesStore.getPackageImages()[
           item.labels.DCOS_PACKAGE_NAME
         ];
+      } else if (
+        ServiceValidatorUtil.isApplicationResponse(item) &&
```

Comment:
It seems to me like you could add this clause to the last else if and lose an entire condition block here:

`if (ServiceValidatorUtil.isFrameworkResponse(item) || ServiceValidatorUtil.isApplicationResponse(item)) {`

Reply:
Yes, I can.